Winapp3/Beta/Winapp3.ini

```diff
@@ -633,25 +631,22 @@ FileKey1=%WinDir%\Prefetch|*.*
 
 [Project64 2.X Cache *]
 Section=Dangerous
-DetectFile1=%ProgramFiles%\Project64 2.0
-DetectFile2=%ProgramFiles%\Project64 2.1
+DetectFile=%ProgramFiles%\Project64 2*

```

Comment:
Is CCleaner able to use wildcards in Detect= now? Last I checked, it couldn't.

Reply:
It works very limited with DetectFile:

**Work:**
DetectFile=%ProgramFiles%\\Test*
DetectFile=%ProgramFiles%\\Te*t1
DetectFile=%ProgramFiles%\\Test1\\program*.exe

**All other combinations don't work:**
DetectFile=%ProgramFiles%\\Test*\\
DetectFile=%ProgramFiles%\\Test*\\program.exe
DetectFile=%ProgramFiles%\\*\\program.exe

It seems that a wildcard works only on the last folder-name or file-name of a path!

Reply:
Winapp2ool is able to identify incorrect wildcard use in DetectFiles

Reply:
Ah OK. Yes, I remember now that it actually works. I wish CCleaner would just support full wild card use in both Detect and DetectFile.